core/templates/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -129,8 +132,13 @@ angular.module('oppia').directive('ckEditor4Rte', [
           scope.uiConfig().startupFocusEnabled !== undefined) {
           startupFocusEnabled = scope.uiConfig().startupFocusEnabled;
         }
+
         // Initialize CKEditor.
         var ck = CKEDITOR.inline(<HTMLElement>(el[0].children[0].children[1]), {
+          // Language configs use default language when undefined.
+          language: scope.uiConfig().activeLanguage,
+          contentsLanguage: scope.uiConfig().activeLanguage,
```

Comment:
What's the difference between these 2 configs?

Reply:
language changes the language of the editor UI, contentsLanguage changes the language of the editor contents.

Reply:
Ah ok, makes sense!